docker-compose.yaml

```
@@ -9,40 +9,32 @@
       restart_policy:
         condition: on-failure
     volumes:
-      - db-data:/ElectronyatShop/Data/
-      - images-data:/ElectronyatShop/wwwroot/Images/
+      - images:/ElectronyatShop/wwwroot/Images/
+    env_file:
+      - .env
     environment:
-        - ASPNETCORE_ENVIRONMENT=Production
-        - SqliteConnection="Data Source=Data/ElectronyatShop_Sqlite.db"
+        - ASPNETCORE_ENVIRONMENT=${ASPNETCORE_ENVIRONMENT}
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 8

(indentation)